frontends/ol-components/src/components/Banner/Banner.tsx

```diff
@@ -30,11 +25,13 @@ const BannerBackground = styled.div<BannerBackgroundProps>(
     backgroundImage: backgroundDim
       ? `linear-gradient(rgba(0 0 0 / ${backgroundDim}%), rgba(0 0 0 / ${backgroundDim}%)), url('${backgroundUrl}')`
       : `url(${backgroundUrl})`,
-    backgroundSize: "cover",
```

Comment:
I think the removal of the background size cover is causing the breakage at the particular width that chris had pointed out.

Reply:
Yea, @mbilalmughal and I had a back and forth about the background. The 140% size rule was for ultrawide screens. I moved the rule to only apply to `lg` and above, and for everything else reset it to default with the position centered and top aligned.